integration_tests/pages/page.ts

```diff
@@ -86,7 +86,7 @@ export default abstract class Page {
   }): void {
     const { course, organisation } = pageWithOrganisationAndCoursePresenter
 
-    cy.get('h2:nth-of-type(1)').then(organisationAndCourseHeading => {
+    cy.get('.govuk-grid-column-two-thirds > h2:first-child').then(organisationAndCourseHeading => {
```

Comment:
Your call but I wonder if a data attribute might be a better choice here: https://docs.cypress.io/guides/references/best-practices#Selecting-Elements

Reply:
Yeah I think this might make it clearer what we're targeting actually - I was thinking all these detailed selectors are getting a bit hard to parse. I think I'll add this a tech debt ticket as it would be good to do this across all the integration tests